services/fastify.js

```diff
@@ -8,7 +8,7 @@ const { http, sink, prop } = require('../');
 class FastifyService {
     constructor({ customSink, port = 4001, logger = console } = {}) {
         const app = fastify({ logger: abslog(logger) });
-        this.sink = customSink || new sink.FS();
+        this.sink = customSink || new sink.GCS();
```

Comment:
how should we default/configure this? I feel like either mem or FS should be the default when working locally but it should be easy enough to switch it over to GCS